Formula/xxh.rb

```diff
@@ -0,0 +1,33 @@
+class Xxh < Formula
+  include Language::Python::Virtualenv
+
+  desc "Bring your favorite shell wherever you go through the ssh"
+  homepage "https://github.com/xxh/xxh"
+  url "https://files.pythonhosted.org/packages/6b/c0/148dbdac379ecf0fa33b90e5ef86e70acdaf35341c7688c0bfcce1ed44b9/xxh-xxh-0.8.2.tar.gz"
+  sha256 "38aff928df11f72f41fdcb775cd75768db7cb7da063f844261b08f78fbe147a8"
+
+  depends_on "python"
```

Comment:
Can this formula work with 3.8?
```suggestion
  depends_on "python@3.8"
```

Reply:
Yes it can, thanks 👍